contrib/openshift/manifests/backstop.yaml

```diff
@@ -0,0 +1,119 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: clair-gc-quaybackstop
+  labels:
+    app: clair
+labels:
+  app: clair
+objects:
+- apiVersion: batch/v1
+  kind: CronJob
+  metadata:
+    name: quaybackstop
+  spec:
+    schedule: '23 1 * * 1'
```

Comment:
:thinking:

Reply:
Think there's a better time to run than `01:23Z`?